cmd/context/delete.go

```diff
@@ -28,12 +29,18 @@ import (
 func DeleteCMD() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "delete",
-		Args:  utils.ExactArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
-		Short: "Delete a context",
+		Args:  utils.MinimumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#delete"),
+		Short: "Delete multiple contexts",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			args[0] = okteto.AddSchema(args[0])
-			args[0] = strings.TrimSuffix(args[0], "/")
-			return Delete(args[0])
+			analytics.TrackDeleteMultipleContexts(len(args))
```

Comment:
```suggestion
			analytics.TrackContextDelete(len(args))
```

I would move this function after the actual delete logic and callit `TrackContextDelete` being consistent with `TrackContextUse` and another one already there. I would set the quantity of deleted contexts a properties rather than a different event. WDYT?

Reply:
Moving after the actual delete logic is a good idea   , does this means that it will only be tracked if there’s no error? 

Yes setting the quantity of deleted contexts as a properties is good! I believed this is inline with the changes I made currently in the `TrackDeleteMultipleContexts` function

Reply:
> Moving after the actual delete logic is a good idea , does this means that it will only be tracked if there’s no error?

If we move it after the logic we could also add a property if the cmd failed